test/integration/router_test.go

```
@@ -233,6 +233,12 @@ func createAndStartRouterContainer(dockerCli *dockerClient.Client, masterIp stri
 		},
 	}
 
+	err = pullIfNotPresent(dockerCli, getRouterImage())
```

Why not just fail hard here?